tools/audit-docs.js

```diff
@@ -55,18 +79,53 @@ function formatScores(scores) {
   return formattedScores;
 }
 
+
+// Launch the light-server to run tests again
+const bundle = process.argv[2];
+const delay = process.argv[3];
```

Comment:
I would love to see us validate this being a number so that if its run locally that we let the user know the delay wasn't applied since `NaN` is treated as `0`

Additionally, we don't need to have the or check in the set timeout because its always going to be a number, defaulting to zero already.

```suggestion
const delay = Number(process.argv[3]);
if (isNaN(delay)) {
  console.error(`The provided value "${process.argv[3]}" should be a number, exiting`);
  process.exit(1);
}
```